fdbserver/storageserver.actor.cpp

```diff
@@ -2800,8 +2856,12 @@ ACTOR Future<Void> update( StorageServer* data, bool* pReceivedUpdate )
 }
 
 ACTOR Future<Void> updateStorage(StorageServer* data) {
+	state std::string waitDescription = format("%s/updateStorage", data->thisServerID.toString().c_str());
```

Comment:
pass this string directly into checkDisabled so we do not construct it when not in simulation